src/Helmet.js

```diff
@@ -134,6 +142,22 @@ const updateTitle = title => {
     document.title = title || document.title;
 };
 
+const updateHtmlAttributes = (attributes) => {
+    const htmlTag = document.getElementsByTagName("html")[0];
+
+    const oldAttributeCount = htmlTag.attributes.length;
+    if (oldAttributeCount) {
+        for (let i = oldAttributeCount - 1; i >= 0; i--) {
+            htmlTag.removeAttribute(htmlTag.attributes[i].name);
```

Comment:
Ya, I couldn't figure a good way to keep track of the helmet attributes here without polluting state. If I get rid of this loop then it would only add/overwrite attributes, but then that misses the case of when you do actually mean to get rid of an attribute.